src/App.tsx

```diff
@@ -1,26 +1,51 @@
+/* eslint-disable max-len */
+import { useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
```

Comment:
Consider using `useAppDispatch` if you have a custom hook for dispatching actions. This can help ensure type safety when dispatching actions in a TypeScript environment.